content/docs/command-reference/import.md

```diff
@@ -91,6 +112,10 @@ from the source repo.
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
+- `-s`, `--store` - [enables remote storage](#enabling-remote-storage) for the
+  import, so it can be pushed and pulled like regular project data (e.g. in
```

Comment:
Maybe link pushed and pulled to the cmd refs?

Reply:
Yeah
```suggestion
  import, so it can be [pushed](/doc/command-reference/push) and 
  [pulled](/doc/command-reference/pull) like regular project data (e.g. in
```